### PR TITLE
Add Injective assets

### PR DESCRIPTION
**Pull request by cryptomanz1**

### CODE REVIEW

File: blockchains/nativeinjective/assets/inj1d5vz0uzwlpfvgwrwulxg6syy82axa58y4fuszd/info.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
+    "name": "Arbitrum",
+    "type": "INJ",
```
Comment (vcoolish) [inline blockchains/nativeinjective/assets/inj1d5vz0uzwlpfvgwrwulxg6syy82axa58y4fuszd/info.json:3]: INJECTIVE

Comment (Goughy1991) [reply blockchains/nativeinjective/assets/inj1d5vz0uzwlpfvgwrwulxg6syy82axa58y4fuszd/info.json]: Pau🤣🃏😂🙏😶🏡

**Review by Goughy1991 — COMMENTED** [review]
Alix brown and white on Zac and s and

**Comment by defisaur** [comment]
@vcoolish which version to set for INJECTIVE tokens?

**Review by vcoolish — APPROVED** [review]

**Comment by vcoolish** [comment]
> @vcoolish which version to set for INJECTIVE tokens?

14

**Review by ZemmySJU — COMMENTED** [review]
INJEKTIF

**Pull request merged**